tools/release/adot-operator-images-mirror/get-operator-tag.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+COLLECTOR_TAG=$(cat go.mod | grep "go.opentelemetry.io/collector " | cut -d " " -f2)
+COLLECTOR_PATCH_VER=${COLLECTOR_TAG: -1}
+COLLECTOR_TAG_WITHOUT_PATCH_VER=$(echo $COLLECTOR_TAG | sed 's/.$//')
```

Comment:
```suggestion
COLLECTOR_TAG_WITHOUT_PATCH_VER=$(echo $COLLECTOR_TAG | sed 's/\..*$//')
```
Is this meant to remove the last `.` and everything following?  If so, it's probably not doing what you expect.  Instead, it's removing the last character.

Reply:
This logic really only needs to remove the last character as `COLLECTOR_TAG_WITHOUT_PATCH_VER` is mostly used in `grep` checks where the extra `.` won't change things - fixed regardless though, for better clarity